src/cocoa_window.m

```diff
@@ -423,6 +423,9 @@ - (BOOL)wantsUpdateLayer
 
 - (id)makeBackingLayer
 {
+    if (!window->ns.layer) {
```

Comment:
GLFW uses the [Allman brace style](https://en.wikipedia.org/wiki/Indent_style#Allman_style) and for single lines leaves out braces altogether.